controllers/core/builder_controller.go

```diff
@@ -174,6 +178,11 @@ func (r *BuilderReconciler) getBuilderResult(builder *openfunction.Builder, buil
 		builder.Status.State = res
 		builder.Status.Reason = reason
 		builder.Status.Message = message
+		if !builder.CreationTimestamp.IsZero() {
+			builder.Status.BuildTime = &metav1.Duration{
+				Duration: metav1.Now().UTC().Sub(builder.CreationTimestamp.UTC()).Truncate(time.Second),
```

Comment:
If we're using `BuildTime` to indicate the time used to build the function, maybe we can use `BuildDuration` instead to be more clear

Reply:
Done